serviceRegistry.go

```diff
@@ -65,7 +65,18 @@ func NewMeasuredService(service *Service) MeasuredService {
 type servicesMap map[string]Service
 type categoriesMap map[string]Category
 
-type ServiceRegistry struct {
+type ServiceRegistry interface {
```

Comment:
In my opinion, this a correct abstraction.
I think that there are a lot of unnecessary exported functions and structs in the whole application source code.